# `wheel` package isn't available in ChromiumOS and it's not really required.
#
# `importlib_resources` isn't available in ChromiumOS but it's only required to
# use nMigen Yosys which, having the "original" Yosys installed, isn't needed.

--- a/setup.py
+++ b/setup.py
@@ -38,10 +38,9 @@ def doc_version():
     #long_description="""TODO""",
     license="BSD",
     python_requires="~=3.6",
-    setup_requires=["wheel", "setuptools", "setuptools_scm"],
+    setup_requires=["setuptools", "setuptools_scm"],
     install_requires=[
         "importlib_metadata; python_version<'3.8'",  # for __version__ and nmigen._yosys
-        "importlib_resources; python_version<'3.9'", # for nmigen._yosys
         "pyvcd~=0.2.2", # for nmigen.pysim
         "Jinja2~=2.11", # for nmigen.build
     ],
